src/RecipesBase.jl

```diff
@@ -333,7 +333,7 @@ end
 """
 You can easily define your own plotting recipes with convenience methods:
 ```
-@userplot type GroupHist
+@userplot struct GroupHist
```

Comment:
Really sorry, I was being imprecise. The docstring is more fundamentally wrong here - it should just be 
```julia
@userplot GroupHist
```
instead of
```julia
@userplot type Grouphist
    args
end
```

Reply:
I guess the problem is more on my side, as i am trying to fix code i don't really understand. I guess i need to take a little more time preparing my PRs in the future.